lib/introspector.js

```diff
@@ -17,6 +17,19 @@ const FOUR_SPACES = ' '.repeat(4);
 const SIGNATURE = 'Signature:';
 const NOTATION_PRIVATE = '#private';
 
+const findMatchingBrace = (str, startBrace, endBrace, startBraceIndex) => {
+  let end = (startBraceIndex || str.indexOf(startBrace)) + 1;
+  for (let count = 1; end < str.length; end++) {
+    const ch = str[end];
+    if (ch === startBrace) count++;
+    else if (ch === endBrace) {
+      count--;
+      if (count === 0) break;
+    }
```

Comment:
Just trying to improve readability
```suggestion
    if (ch === startBrace) {
      count++;
      continue;
    }
    if (ch === endBrace) {
      count--;
      if (count === 0) break;
    }
```

Reply:
PTAL

Reply:
That's fine